fullstack/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# frontend/Dockerfile
+# Use an official Node.js runtime as a parent image
+FROM node:20.13.1
+
+# Set the working directory in the container
+WORKDIR /Cloudy-with-a-Chance-of-LoRa/fullstack
+
+# Copy the package.json and package-lock.json files
+COPY /package*.json ./
+
+# Install NVM and Node.js
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash && \
+    export NVM_DIR="$HOME/.nvm" && \
+    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" && \
+    nvm install 20.13.1 && \
+    nvm use 20.13.1 && \
+    nvm alias default 20.13.1 && \
+    npm install pg pg-promise && \
+    npm i dotenv
+
+# Copy the rest of the application code
+COPY . .
```

Comment:
just a question since im not good with docker. but copying the whole code with out techs from dockers its self seems like it could be an issue in the future.